CHANGELOG.md

```diff
@@ -0,0 +1,86 @@
+ <a name="1.0.0"></a>
+# 1.0.0 (2018-03-10)
```

Comment:
why it is shown as 1.0.0 ?

Reply:
Actually, **1.0.0** is the version specified in *package.json*. Do I increase the version in *package.json*?